src/JuMP.jl

```diff
@@ -296,9 +300,11 @@ end
 
 Return mode (DIRECT, AUTOMATIC, MANUAL) of model.
 """
-mode(model::Model) = moi_mode(backend(model))
-# The type of backend(model) is unknown so we directly redirect to another
-# function.
+function mode(model::Model)
+    # The type of `backend(model)` is unknown so we directly redirect to another
```

Comment:
Nit nit: Even though the comment was already there, `directly redirect` sounds weird. `so we redirect to` is fine.